### ISSUE TITLE
[Website][Doc] Gluon Loss functions LaTex rendering issue

### ISSUE THREAD

**Issue by ehsanmok** [apache/incubator-mxnet#17424]
https://mxnet.apache.org/api/python/docs/tutorials/packages/gluon/loss/loss.html#Cross-Entropy-Loss-with-Softmax

Seems this page doesn't not render latex

**Comment by ehsanmok** [comment]
@mxnet-label-bot add [Doc, Good First Issue]

**Comment by TEChopra1000** [comment]
This issue was resolved and this ticket can be closed.

**Comment by TEChopra1000** [comment]
PR: https://github.com/apache/incubator-mxnet/pull/17435
